packages/dashboards/src/lib/components/timeseries-widget/transformer/loess.ts

```diff
@@ -0,0 +1,283 @@
+// Based on org.apache.commons.math.analysis.interpolation.LoessInterpolator
```

Comment:
we should also include the comment header to newly added the source files (keep these, but prepend the SWI text as seen in other sourcefiles)

(did we check the original license in these? just to be sure)